### ISSUE TITLE
Unable to setup

### ISSUE THREAD

**Issue by mjanser** [black-sheep-dev/harbour-quartermaster#18]
I'm unable to setup this app with the latest version. On the second screen I see the message "Seite konnte nicht geladen werden" (site could not be loaded).

When starting the app in the terminal, I get the following output:
```
nemo@Sailfish ~]$ harbour-quartermaster
[D] unknown:0 - Using Wayland-EGL
[W] unknown:9 - file:///usr/share/harbour-quartermaster/qml/harbour-quartermaster.qml:9:30: QML OverviewPage: Bi
nding loop detected for property "status"
[W] unknown:1283 - file:///usr/lib/qt5/qml/Sailfish/Silica/PageStack.qml:1283:13: QML AnimatedLoader: (file:///u
sr/share/harbour-quartermaster/qml/pages/wizard/WizardConnectionPage.qml:77:22: Cannot assign a value to a signa
l (expecting a script to be run)
EnterKey.onClicked: false
^)
```

**Comment by black-sheep-dev** [comment]
Thanks for your report! It is fixed with version 0.2.5.

**Comment by mjanser** [comment]
Thank you for fixing this so fast, it works now!
